### ISSUE TITLE
async + arrow-parens

### ISSUE THREAD

**Issue by ChALkeR** [babel/babel-eslint#310]
```js
async (a, b) => {}
```
results in `Expected parentheses around arrow function argument` error.

**Comment by hzoo** [comment]
Dup of https://github.com/babel/babel-eslint/issues/268, https://github.com/babel/babel-eslint/issues/252

Use https://github.com/babel/eslint-plugin-babel#rules

`babel/arrow-parens: Handles async functions correctly`

**Comment by ChALkeR** [comment]
Thanks!
I checked only open issues for some reason…

**Comment by hzoo** [comment]
Np, If there is an incompatibility with an existing rule + experimental feature - we might need to modify the rule to account for it like in this case
